src/core/sys/darwin/netinet/in_.d

```diff
@@ -0,0 +1,605 @@
+    enum IPPROTO_CPHB      = 73;
+    enum IPPROTO_WSN       = 74;
+    enum IPPROTO_PVP       = 75;
+    enum IPPROTO_BRSATMON  = 76;
+    enum IPPROTO_ND        = 77;
+    enum IPPROTO_WBMON     = 78;
+    enum IPPROTO_WBEXPAK   = 79;
+    enum IPPROTO_EON       = 80;
+    enum IPPROTO_VMTP      = 81;
+    enum IPPROTO_SVMTP     = 82;
+    enum IPPROTO_VINES     = 83;
+    enum IPPROTO_TTP       = 84;
+    enum IPPROTO_IGP       = 85;
+    enum IPPROTO_DGP       = 86;
+    enum IPPROTO_TCF       = 87;
+    enum IPPROTO_IGRP      = 88;
+    enum IPPROTO_OSPFIGP   = 89;
+    enum IPPROTO_SRPC      = 90;
+    enum IPPROTO_LARP      = 91;
+    enum IPPROTO_MTP       = 92;
+    enum IPPROTO_AX25      = 93;
+    enum IPPROTO_IPEIP     = 94;
+    enum IPPROTO_MICP      = 95;
+    enum IPPROTO_SCCSP     = 96;
+    enum IPPROTO_ETHERIP   = 97;
+    enum IPPROTO_ENCAP     = 98;
+    enum IPPROTO_APES      = 99;
+    enum IPPROTO_GMTP      = 100;
+    enum IPPROTO_PIM       = 103;
+    enum IPPROTO_IPCOMP    = 108;
+    enum IPPROTO_PGM       = 113;
+    enum IPPROTO_SCTP      = 132;
+    enum IPPROTO_DIVERT    = 254;
+}
+enum IPPROTO_RAW = 255;
+
+static if (_DARWIN_C_SOURCE)
+{
+    enum IPPROTO_MAX  = 256;
+    enum IPPROTO_DONE = 257;
+}
+
+static if (_DARWIN_C_SOURCE)
+{
+    enum IPPORT_RESERVED = 1024;
+    enum IPPORT_USERRESERVED = 5000;
+
+    enum IPPORT_HIFIRSTAUTO = 49152;
+    enum IPPORT_HILASTAUTO  = 65535;
+
+    enum IPPORT_RESERVEDSTART = 600;
+}
+
+static if (_DARWIN_C_SOURCE)
+{
+    extern(D) bool IN_CLASSA(in_addr_t i) pure @safe { return (i & 0x80000000) == 0; }
+    enum IN_CLASSA_NET    = 0xff000000;
+    enum IN_CLASSA_NSHIFT = 24;
+    enum IN_CLASSA_HOST   = 0x00ffffff;
+    enum IN_CLASSA_MAX    = 128;
+
+    extern(D) bool IN_CLASSB(in_addr_t i) pure @safe { return (i & 0xc0000000) == 0x80000000; }
+    enum IN_CLASSB_NET    = 0xffff0000;
+    enum IN_CLASSB_NSHIFT = 16;
+    enum IN_CLASSB_HOST   = 0x0000ffff;
+    enum IN_CLASSB_MAX    = 65536;
+
+    extern(D) bool IN_CLASSC(in_addr_t i) pure @safe { return (i & 0xe0000000) == 0xc0000000; }
+    enum IN_CLASSC_NET    = 0xffffff00;
+    enum IN_CLASSC_NSHIFT = 8;
+    enum IN_CLASSC_HOST   = 0x000000ff;
+
+    extern(D) bool IN_CLASSD(in_addr_t i) pure @safe { return (i & 0xf0000000) == 0xe0000000; }
+    enum IN_CLASSD_NET    = 0xf0000000;
+    enum IN_CLASSD_NSHIFT = 28;
+    enum IN_CLASSD_HOST   = 0x0fffffff;
+    extern(D) bool IN_MULTICAST(in_addr_t i) pure @safe { return IN_CLASSD(i); }
+
+    // The fact that these are identical looks suspicious (they're not quite
+    // identical on Linux). However, this _is_ how they're defined in FreeBSD
+    // and on OS X. So, while it _might_ be a bug, if it is, it's an upstream
+    // one, and we're compatible with it.
+    extern(D) bool IN_EXPERIMENTAL(in_addr_t i) pure @safe { return (i & 0xf0000000) == 0xf0000000; }
+    extern(D) bool IN_BADCLASS(in_addr_t i) pure @safe { return (i & 0xf0000000) == 0xf0000000; }
+
```

Comment:
Because they're in `core.sys.posix.netinet._in`, because they're POSIX, unlike the stuff in this PR.